graylog2-web-interface/src/views/pages/ShowDashboardInBigDisplayMode.jsx

```diff
@@ -64,4 +62,4 @@ const ShowDashboardInBigDisplayMode = ({ location, params, route, view: { view,
   );
 };
 
-export default withRouter(connect(ShowDashboardInBigDisplayMode, { view: ViewStore, viewMetadata: ViewMetadataStore, query: CurrentQueryStore }));
```

Comment:
I removed  the `ViewMetadataStore` and `CurrentQueryStore ` usage, because it looks like we are not using `viewMetadata` and `query`.